.github/workflows/frontend_tests.yml

```diff
@@ -21,6 +21,18 @@ jobs:
         with:
           python-version: '2.x'
           architecture: 'x64'
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-oppia-dependencies
+        with:
+          path: |
+            ../oppia_tools/
+            node_modules/
+            third_party/
+            ~/.cache/pip
+            ~/.cache/TravisChrome/
+          key: ${{ runner.os }}-build-${{ env.cache-name }}
```

Comment:
Few questions @DubeySandeep:
1. Why are we caching stuff used by e2e tests in the "frontend_tests" workflow?
2. Are caches shared across builds or does each workflow store/use its own cache (e.g. workflow store the cache on the first run and uses it on subsequent runs)?
3. Did you take a look at [this](https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy)? It says there is a cache limit of 5GB per repo, this means that when we exceed the limit, the new paths are cached and old ones get evicted until the total size is <5GB. Has this been considered? We will easily exceed this limit with just 2 builds considering our current cache sizes (screenshot below). This means that when we restart older builds, it will have to re-download deps anyway:
![2020-11-17](https://user-images.githubusercontent.com/11008603/100154159-2b063d00-2ecb-11eb-9d3d-cc91adc28b41.png).
4. Why are we caching "TravisChrome" on actions? :)